src/main/kotlin/org/rust/ide/inspections/import/AutoImportFix.kt

```diff
@@ -173,6 +187,44 @@ class AutoImportFix(path: RsPath) : LocalQuickFixOnPsiElement(path), HighPriorit
             return Context(basePath, candidates)
         }
 
+        fun findApplicableContext(project: Project, methodCall: RsMethodCall): Context<Unit>? {
+            val results = methodCall.reference.multiResolve()
+            if (results.isEmpty()) return Context(Unit, emptyList())
+
+            val lookup = ImplLookup.relativeTo(methodCall)
+            val mod = methodCall.containingMod
```

Comment:
Why do we skip scopes to the mod declaration? I can imagine an example where it is wrong:
```rust
mod foo {
    fn foo(a: A) {
        use bar::Baz;
        a.baz(); // baz defined on Baz trait that is implemented for `A` type
    }
}
```